src/main/java/io/strimzi/kafka/quotas/StaticQuotaConfig.java

```diff
@@ -76,7 +78,7 @@ public StaticQuotaConfig(Map<String, ?> props, boolean doLog) {
                         .define(PRODUCE_QUOTA_PROP, DOUBLE, Double.MAX_VALUE, HIGH, "Produce bandwidth rate quota (in bytes)")
                         .define(FETCH_QUOTA_PROP, DOUBLE, Double.MAX_VALUE, HIGH, "Consume bandwidth rate quota (in bytes)")
                         .define(REQUEST_QUOTA_PROP, DOUBLE, Double.MAX_VALUE, HIGH, "Request processing time quota (in seconds)")
-                        .define(EXCLUDED_PRINCIPAL_NAME_LIST_PROP, LIST, List.of(), MEDIUM, "List of principals that are excluded from the quota")
+                        .define(EXCLUDED_PRINCIPAL_NAME_LIST_PROP, STRING, "", MEDIUM, "List of principals that are excluded from the quota")
```

Comment:
Why empty string for the default value?  Why not null?

Reply:
I wanted to have default to empty String, as the split can cause NPE in case of null, or?

Reply:
yes, your way functions, but it is weird.  The effect will be the excludedPrinicipal list ends up a singleton list with an empty string as a member.  Better to handle the null explicitly and have that produce an empty list.

Reply:
Yeah, makes sense. Good catch :) thanks